cypress/support/cypress-support/src/utils.js

```diff
@@ -616,6 +616,89 @@ function checkForTags(tags) {
   }
 }
 
+/**
+ * FireLog class provides assertion methods with logging using Cypress's cy.log().
+ * It wraps Cypress's assertion methods, allowing logging of messages for each assertion.
+ * @class
+ *
+ * @example
+ * // Usage example
+ * fireLog.isNotNull(someValue, "Some message", "Log message");
```

Comment:
Why do we need 2 message fields? Can't `Some message` and `Log message` be the same?

Reply:
updated